app/controllers/properties_controller.rb

```diff
@@ -4,6 +4,7 @@ class PropertiesController < ApplicationController
 
   def show
     @property_details = Hackney::Property.find(params[:ref])
+    @cautionary_contact = Hackney::CautionaryContact.find(@property_details.reference)
```

Comment:
We should make a `find_by_property_reference` method. Otherwise I would expect it to follow ActiveRecord convention and receive a `cautionary_contact_id` (which may not even exist).